apps/demo-react/components/ConfigContextProviders.tsx

```diff
@@ -34,23 +33,38 @@ const connectors = getConnectors({
 
 const client = createClient({
   connectors,
-  autoConnect: true,
+  autoConnect: false, // default wagmi autoConnect should be false
   provider,
   webSocketProvider,
 });
 
-const ProviderWeb3WithProps = ({ children }: { children: React.ReactNode }) => {
+const ConfigContextProviders = ({
+  children,
+}: {
+  children: React.ReactNode;
+}) => {
+  const defaultChain =
+    supportedChains.find((chain) => chain.id === defaultChainId) ||
+    supportedChains[0];
+
   return (
     <WagmiConfig client={client}>
-      <ProviderWeb3
-        defaultChainId={defaultChainId}
-        supportedChainIds={supportedChainsIds}
+      <ReefKnot
         rpc={rpcUrlsString}
+        chains={supportedChains}
+        defaultChain={defaultChain}
         walletconnectProjectId={WC_PROJECT_ID}
+        autoConnect
       >
-        {children}
-      </ProviderWeb3>
+        <ProviderSDKWithProps
```

Comment:
Why do we need to continue using this provider?

Reply:
Because the app is still using hooks from `@lido-sdk/react`, which requires `ProviderSDK` to be set.